modules/log/console_windows.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package log
+
+import "golang.org/x/sys/windows"
+
+const ENABLE_VIRTUAL_TERMINAL_PROCESSING = 0x0004
```

Comment:
This name is not encouraged. Maybe name it `EnableVirtual...` ?

Reply:
Done